[java-9(10)] java.lang.ExceptionInInitializerError: null

Environment:
play: 2.6.20
scala: 2.12.7
play2-maven-plugin: 1.0.0-rc3

do `mvn clean install`
After moving to java-9(10) get the underlying error:

```
[INFO] --- play2-maven-plugin:1.0.0-rc3:routes-compile (default-routes-compile) @ server ---
[INFO] Generating injected router
[INFO] 1 router processed, 1 compiled
[INFO] 
[INFO] --- play2-maven-plugin:1.0.0-rc3:template-compile (default-template-compile) @ server ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11.728 s
[INFO] Finished at: 2018-11-02T11:17:00Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.google.code.play2-maven-plugin:play2-maven-plugin:1.0.0-rc3:template-compile (default-template-compile) on project server: Execution default-template-compile of goal com.google.code.play2-maven-plugin:play2-maven-plugin:1.0.0-rc3:template-compile failed: An API incompatibility was encountered while executing com.google.code.play2-maven-plugi
n:play2-maven-plugin:1.0.0-rc3:template-compile: java.lang.ExceptionInInitializerError: null
[ERROR] -----------------------------------------------------
[ERROR] realm =    extension>com.google.code.play2-maven-plugin:play2-maven-plugin:1.0.0-rc3
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/oleg/.m2/repository/com/google/code/play2-maven-plugin/play2-maven-plugin/1.0.0-rc3/play2-maven-plugin-1.0.0-rc3.jar
[ERROR] urls[1] = file:/home/oleg/.m2/repository/com/google/code/play2-maven-plugin/play2-provider-api/1.0.0-rc3/play2-provider-api-1.0.0-rc3.jar
[ERROR] urls[2] = file:/home/oleg/.m2/repository/com/google/code/play2-maven-plugin/play2-source-position-mappers/1.0.0-rc3/play2-source-position-mappers-1.0.0-rc3.jar
[ERROR] urls[3] = file:/home/oleg/.m2/repository/com/google/code/play2-maven-plugin/play2-source-watcher-api/1.0.0-rc3/play2-source-watcher-api-1.0.0-rc3.jar
[ERROR] urls[4] = file:/home/oleg/.m2/repository/com/google/code/sbt-compiler-maven-plugin/sbt-compiler-api/1.0.0/sbt-compiler-api-1.0.0.jar
[ERROR] urls[5] = file:/home/oleg/.m2/repository/org/apache/ant/ant/1.9.4/ant-1.9.4.jar
[ERROR] urls[6] = file:/home/oleg/.m2/repository/org/apache/ant/ant-launcher/1.9.4/ant-launcher-1.9.4.jar
[ERROR] urls[7] = file:/home/oleg/.m2/repository/org/apache/maven/shared/maven-common-artifact-filters/1.4/maven-common-artifact-filters-1.4.jar
[ERROR] urls[8] = file:/home/oleg/.m2/repository/org/codehaus/plexus/plexus-archiver/3.6.0/plexus-archiver-3.6.0.jar
[ERROR] urls[9] = file:/home/oleg/.m2/repository/org/codehaus/plexus/plexus-io/3.0.1/plexus-io-3.0.1.jar
[ERROR] urls[10] = file:/home/oleg/.m2/repository/commons-io/commons-io/2.6/commons-io-2.6.jar
[ERROR] urls[11] = file:/home/oleg/.m2/repository/org/apache/commons/commons-compress/1.16.1/commons-compress-1.16.1.jar
[ERROR] urls[12] = file:/home/oleg/.m2/repository/org/objenesis/objenesis/2.6/objenesis-2.6.jar
[ERROR] urls[13] = file:/home/oleg/.m2/repository/org/iq80/snappy/snappy/0.4/snappy-0.4.jar
[ERROR] urls[14] = file:/home/oleg/.m2/repository/org/tukaani/xz/1.8/xz-1.8.jar
[ERROR] urls[15] = file:/home/oleg/.m2/repository/org/codehaus/plexus/plexus-utils/3.1.0/plexus-utils-3.1.0.jar
[ERROR] urls[16] = file:/home/oleg/.m2/repository/org/sonatype/plexus/plexus-build-api/0.0.7/plexus-build-api-0.0.7.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] : object java.lang.Object in compiler mirror not found.
[ERROR] -> [Help 1]
```

For the build i use this configuration for maven compiler plugin:
```
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.0</version>
                <configuration>
                    <source>10</source>
                    <target>10</target>
                    <encoding>UTF-8</encoding>
                    <compilerArgs>
                        <arg>--add-modules</arg><arg>java.xml.bind</arg>
                        <arg>--add-exports</arg><arg>java.xml.bind=ALL-UNNAMED</arg>
                        <arg>--illegal-access</arg><arg>permit</arg>
                    </compilerArgs>
                </configuration>
            </plugin>
 ```

--- a/play2-providers/play2-provider-play26/pom.xml
+++ b/play2-providers/play2-provider-play26/pom.xml
@@ -73,7 +73,7 @@ under the License.
 
         <dependency>
             <groupId>com.typesafe.play</groupId>
-            <artifactId>sbt-routes-compiler_2.10</artifactId>
+            <artifactId>sbt-routes-compiler_2.12</artifactId>
             <version>${play2.version}</version>
             <exclusions>
                 <exclusion>
@@ -91,7 +91,7 @@ under the License.
 
         <dependency>
             <groupId>com.typesafe.play</groupId>
-            <artifactId>twirl-compiler_2.10</artifactId>
+            <artifactId>twirl-compiler_2.12</artifactId>
             <version>${twirl.version}</version>
         </dependency>
 

--- a/play2-providers/play2-provider-play26/src/main/java/com/google/code/play2/provider/play26/Play26TemplateCompiler.java
+++ b/play2-providers/play2-provider-play26/src/main/java/com/google/code/play2/provider/play26/Play26TemplateCompiler.java
@@ -24,7 +24,7 @@ import java.util.Collections;
 import java.util.List;
 
 import scala.Option;
-import scala.collection.JavaConversions;
+import scala.collection.JavaConverters;
 import scala.collection.Seq;
 import scala.io.Codec;
 
@@ -93,7 +93,7 @@ public class Play26TemplateCompiler
 
     private List<String> getDefaultImports( List<String> playImports )
     {
-        List<String> twirlDefaultImports = JavaConversions.asJavaList( TwirlCompiler.DefaultImports() );
+        List<String> twirlDefaultImports = JavaConverters.seqAsJavaList( TwirlCompiler.DefaultImports() );
         List<String> defaultImports = new ArrayList<String>( twirlDefaultImports.size() + playImports.size() );
         defaultImports.addAll( twirlDefaultImports );
         defaultImports.addAll( playImports );
@@ -120,7 +120,7 @@ public class Play26TemplateCompiler
             String formatterType = formatterTypes[index];
             Seq<String> additionalImportsSeq = getAdditionalImports( ext );
             Seq<String> constructorAnnotationsSeq =
-                JavaConversions.asScalaBuffer( Arrays.asList( constructorAnnotations ) ).toSeq();
+                JavaConverters.asScalaBuffer( Arrays.asList( constructorAnnotations ) ).toSeq();
             try
             {
                 Option<File> resultOption =
@@ -158,7 +158,7 @@ public class Play26TemplateCompiler
         {
             formattedAdditionalImports.add( additionalImport.replace( "%format%", format ) );
         }
-        return JavaConversions.asScalaBuffer( formattedAdditionalImports ).toSeq();
+        return JavaConverters.asScalaBuffer( formattedAdditionalImports ).toSeq();
     }
 
 }

